src/addItem.js

```diff
@@ -0,0 +1,24 @@
+import displayTodos from './displayItems.js';
+import attachEventListeners from './attachEventListener.js';
+
+const addTodo = () => {
+  const input = document.querySelector('.input-field');
+  input.addEventListener('keypress', (e) => {
+    if (e.key === 'Enter') {
+      if (input.value === '') {
+        // eslint-disable-next-line no-alert
+        alert('You cannot add empty fields');
+      } else {
+        const todos = JSON.parse(localStorage.getItem('todo') || '[]');
+        const id = Math.random().toString(36).substr(0, 5);
```

Comment:
- [ ] New entries **do not match** the property index. The _property index_ must be set to the value of the _new array length_ (i.e. if a 5th task is added to the list, the index of that task should equal 5).

**Index mismatch in your application:**
![image](https://user-images.githubusercontent.com/12745474/135218755-d5f4ae5a-c587-49d6-a54f-8af715295360.png)

Reply:
Hi @rloterh , I am populating an index based on a random number and converting it to a string but I will populate my id as you have recommended based on the array length.

Reply:
This issue is well resolved! ✔️

Reply:
Yeey! Thank you for the timely review!